statsd/aggregator.go

```diff
@@ -318,7 +315,7 @@ func (a *MetricAggregator) receiveSet(name, tags string, value string, now time.
 	}
 }
 
-// receiveMetric is called for each incoming metric on MetricChan.
```

Comment:
I don't think this is needed, is it? I'd rather keep it on the MetricAggregator for consistency. Think it this way: Ii this was an interface, it would be part of the behaviour.